src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs

```diff
@@ -203,7 +204,7 @@ public ITempDataDictionary TempData
 
             if (htmlAttributes != null)
             {
-                foreach (var helper in HtmlAttributePropertyHelper.GetProperties(htmlAttributes))
+                foreach (var helper in HtmlAttributePropertyHelper.GetProperties(htmlAttributes.GetType().GetTypeInfo()))
```

Comment:
Not if I make this change https://github.com/aspnet/Mvc/pull/6276/files#diff-bde884ef326029a4210b64620a71026fR16 Should I change the parameter to `Type type` instead?